core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -994,6 +994,11 @@ public static String getSyncMetricName(long mountId) {
           .setDescription("The number of failed Load commands")
           .setMetricType(MetricType.COUNTER)
           .build();
+  public static final MetricKey MASTER_JOB_LOAD_RUNNING =
```

Comment:
maybe change to JOB_RUNNING since scheduler now is more generic

Reply:
Thanks for the suggestion! I renamed it to `MASTER_JOB_SCHEDULER_RUNNING`.